ros2_control_test_assets/include/ros2_control_test_assets/descriptions.hpp

```diff
@@ -113,6 +113,136 @@ const auto urdf_head =
       </geometry>
     </collision>
   </link>
+  <joint name="joint3" type="revolute">
+    <origin rpy="1.57079632679 0 0" xyz="0 0 0.9"/>
+    <parent link="link2"/>
+    <child link="link3"/>
+    <limit effort="0.1" lower="-3.14159265359" upper="3.14159265359" velocity="0.2"/>
+  </joint>
+  <link name="link3">
+    <inertial>
+      <mass value="0.01"/>
+      <origin xyz="0 0 0"/>
+      <inertia ixx="0.001" ixy="0.0" ixz="0.0" iyy="0.001" iyz="0.0" izz="0.001"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <cylinder length="1" radius="0.1"/>
+      </geometry>
+      <material name="DarkGrey">
+        <color rgba="0.4 0.4 0.4 1.0"/>
+      </material>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <cylinder length="1" radius="0.1"/>
+      </geometry>
+    </collision>
+  </link>
+  <joint name="tool_joint" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 1"/>
+    <parent link="link2"/>
+    <child link="tool_link"/>
+  </joint>
+  <link name="tool_link">
+  </link>
+)";
+const auto urdf_head_mimic =
```

Comment:
@christophfroehlich Just to be sure, for this there is no modified `<ros2_control>`, because you wanted to test the default case of `mimic=true` when not defined. Am I right?

Reply:
I had to add this for the generic system tests, e.g.  `hardware_system_2dof_with_mimic_joint`, which hadn't have mimic definition in the URDF but only in the `ros2_control` tag

Reply:
Perfect. Thanks for the explanation.